webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading-comment.js

```diff
@@ -128,12 +125,14 @@ export class SakaiRubricGradingComment extends RubricsElement {
         height: 40
       });
 
-      commentEditor.on('change', () => this.criterion.comments = commentEditor.getData());
       commentEditor.on('blur', () => {
 
-        // When we click away from the comment editor we need to save the comment
-        this.criterion.comments = commentEditor.getData();
-        this.dispatchEvent(this._getUpdateCommentEvent());
+        // When we click away from the comment editor we need to save the comment, but only if the comment has been updated
+        let updatedComments = commentEditor.getData();
+        if (this.criterion.comments != updatedComments) {
```

Comment:
Now we only update `this.criterion.comments` var and run `_getUpdateCommentEvent` event if comments were really updated.